jsonschema/exceptions.py

```diff
@@ -61,14 +61,18 @@ def __init__(
     def __repr__(self):
         return "<%s: %r>" % (self.__class__.__name__, self.message)
 
-    def __unicode__(self):
+    def _formatted_message(self, formatter=None):
         essential_for_verbose = (
             self.validator, self.validator_value, self.instance, self.schema,
         )
         if any(m is _unset for m in essential_for_verbose):
             return self.message
 
-        pschema = pprint.pformat(self.schema, width=72)
+        if (callable(formatter)):
```

Comment:
Would prefer to just always have formatter take a callable -- if this is just to handle the `None` case, just using `formatter=pformat` or `partial(pformat, width=72)` should make it go away I think?

Reply:
Ah, so use that in place of the `formatter=None` argument in the declaration?

I put the `callable()` check there in case a non-callable formatter gets passed in.  It seems safer to me.

Reply:
I'll replace `formatter=None` and the other logic with `formatter=partial(pformat, width=72)`, even though checking with `callable()` seems safer to me.  If you change your mind, tell me and I can put it back the other way.